spec/controllers/v1/people_controller_spec.rb

```diff
@@ -1,15 +1,17 @@
 require 'rails_helper'
 
 describe V1::PeopleController,  type: :controller do
-
   describe "POST create" do
     it "returns success" do
-      post :create, {email: 'dude@gmail.com'}
+      user = instance_double("Person", id: 3)
+      expect(Person).to receive(:create_or_update).
+        with(email: 'user@example.com', tags: ['test']) { user }
+      post :create, person: { email:'user@example.com', tags: ['test'] }
       json_response = JSON.parse(response.body)
 
       expect(response).to be_success
       expect(json_response).to have_key('id')
-      expect(json_response['id']).to eq(57126)
+      expect(json_response['id']).to eq(3)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.